guides/common/modules/proc_disabling-puppet.adoc

```diff
@@ -13,6 +13,10 @@ To discontinue using Puppet in your {Project}, follow this procedure.
 ----
 # {foreman-maintain} plugin purge-puppet
 ----
+. Refresh the {Smartproxy} features:
+.. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
+.. For each {SmartProxy}, select *Refresh* from the *Actions* list.
+.. Optional: Verify that *Puppet* and *Puppet CA* are gone from the list of features after clicking on a {SmartProxy} name.
```

Comment:
```suggestion
. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
. For each {SmartProxy}, select *Refresh* from the *Actions* list.
. Optional: Verify that *Puppet* and *Puppet CA* are gone from the list of features after clicking on a {SmartProxy} name.
```

Maybe we can go without nested lists here?

Reply:
Applied :+1: